### PR TITLE
added portions to the Json

### PR DESCRIPTION
**Pull request by DeemoONeill**
added a recipe class to the src file and changed the collection of the recipes updated the testing file to work with the new collection and class

### CODE REVIEW

File: src/recipes.py [hunk 1]
```diff
@@ -5,8 +5,34 @@
...
+        # pasta can be divisible by any number
+        self.ingredients = normalised_ingredients
+
+        self.portions = portions
+
+        def ingredient_quantity(portions_to_cook):
+            if portions_to_cook == self.portions:
+                return self.portions
+            else:
+                return self.ingredients * portions_to_cook
```
Comment (DeemoONeill) [inline src/recipes.py]: Should this be 

```python
return {ingredient : quantity*portions_to_cook for ingredient, quantity in self.normalised_ingredients.items()}

```
?

Comment (DeemoONeill) [reply src/recipes.py]: @Rustyoneill 

I did suggest this fix while i was away last weekend :P

**Comment by DeemoONeill** [comment]
@Rustyoneill give this a review, get familiar. I'll merge it either tomorrow or saturday

**Review by DeemoONeill — COMMENTED** [review]
lgtm

**Review by Rustyoneill — APPROVED** [review]
lgtm

**Pull request merged**